utils/functions.py

```diff
@@ -86,9 +68,9 @@ def search(list_to_search: list, value, key, cutoff=5, return_key=False, strict=
     if result is None:
         partial_matches = [a for a in list_to_search if value.lower() in key(a).lower()]
         if len(partial_matches) > 1 or not partial_matches:
-            names = [key(d) for d in list_to_search]
-            fuzzy_map = {key(d): d for d in list_to_search}
-            fuzzy_results = [r for r in process.extract(value, names, scorer=fuzz.ratio) if r[1] >= cutoff]
+            names = [key(d).lower() for d in list_to_search]
```

Comment:
It turns out some scores returned by `fuzz.ratio` were lower than they should have been due to case sensitivity - this fixes that